cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -54,16 +54,16 @@ def test_schema_definition(self):
             self.assertTrue("type" in self.schema_def["components"]["obs"]["columns"][i])
             if i == "curie":
                 self.assertIsInstance(
-                    self.schema_def["components"]["obs"]["columns"][i]["curie_constrains"],
+                    self.schema_def["components"]["obs"]["columns"][i]["curie_constraints"],
                     dict,
                 )
                 self.assertIsInstance(
-                    self.schema_def["components"]["obs"]["columns"][i]["curie_constrains"]["ontolgies"],
+                    self.schema_def["components"]["obs"]["columns"][i]["curie_constraints"]["ontolgies"],
                     list,
                 )
 
                 # Check that the allowed ontologies are in the ontology checker
-                for ontology_name in self.schema_def["components"]["obs"]["columns"][i]["curie_constrains"][
+                for ontology_name in self.schema_def["components"]["obs"]["columns"][i]["curie_constraints"][
```

Comment:
shouldn't this typo have resulted in failing tests elsewhere?

Reply:
It for sure should have. I didn't mean to push this up with this change (I was looking into it separately)--I still don't know why it didn't fail previously...going to keep looking into it and address it in a different PR.